include/hip/hcc_detail/grid_launch_GGL.hpp

```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 #include "concepts.hpp"
 #include "helpers.hpp"
 
-#include "hc.hpp"
+#include "hcc/hc.hpp"
```

Comment:
The problem with this is if you build and install hcc from source using ```cmake <...>; make; make install```, hcc headers are installed in <hcc_install_root>/include and not <hcc_install_root>/include/hcc. Hence this change breaks building hip unless hcc is installed using deb or rpm packages.